apps/ledger-live-desktop/src/renderer/components/OperationsList/index.tsx

```diff
@@ -151,6 +152,7 @@ export class OperationsList extends PureComponent<Props, State> {
                       return null;
                     }
                   }
+                  const mainAccount = getMainAccount(account, parentAccount);
```

Comment:
I think the change in this file (this mainAccount line) is not necessary because the "isEditableOperation" is already able to handle both token account and normal account.

Reply:
`isEditableOperation` takes an `Account` as param but `account` is an `AccountLike` so we need to get the associated main account.

cf.
https://github.com/LedgerHQ/ledger-live/blob/a12f49718a7451892610b442136c06583469dddc/libs/coin-framework/src/operation.ts#L243-L269